Tasks/Aleksiutovich/Task7/logic_b.py

```diff
@@ -0,0 +1,78 @@
+import json
+from abc import ABC, abstractmethod
+
+
+# Этот код написан с помощью ChatGPT4.

```

Comment:
yeah, I can see) it suggested totally unnecessary ABC class FilePath, but the search mechanism in Collection is good. It's better to take a suggestion from an AI with a huge grain of salt, especially when you do not know the exact answer to your question.